modules/gdscript/gdscript.h

```diff
@@ -133,7 +133,7 @@ class GDScript : public Script {
 
 #endif
 
-	bool _update_exports();
+	bool _update_exports(bool *p_ret_err = (bool *)0, bool p_recursive_call = false);
```

Comment:
Does it make a difference to use `(bool *)0` here instead of simply `nullptr`?

Reply:
No now I've changed it to `nullptr`, it was opened back then where we don't use `nullptr`